templates/python-selenium/README.md

```diff
@@ -1,5 +1,20 @@
-# Selenium Actor template
+# Python - Selenium & Chrome Actor template
```

Comment:
I don't think it needs to have the `Python` prefix here, we don't have that pattern for other templates. In CLI you need to choose the language so you already know it's Python templates and on template detail we have it the category in the sidebar.

```suggestion
# Selenium & Chrome Actor template
```